platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/parser/PomReader.java

```diff
@@ -341,11 +343,20 @@ public String getParentVersion() {
     }
 
     public String getPackaging() {
-        String val = getFirstChildText(projectElement, PACKAGING);
+        String val = replaceProps(getFirstChildText(projectElement, PACKAGING));
```

Comment:
💅 Because of confusion with Kotlin, Lombok and JEPs, and because it is carries little meaning, I think `val` shouldn't be used as a variable name.  Why not use just `packaging`?